httpbp/middlewares.go

```diff
@@ -116,13 +126,14 @@ func StartSpanFromTrustedRequest(
 // NewBaseplateServer function which will automatically include InjectServerSpan
 // as one of the Middlewares to wrap your handlers in.
```

Comment:
Please also update the doc of this function to mention that we auto apply `HTTPErrorSuppressor` here.